dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,321 @@
+                   + "            WHEN   45 THEN 'BOOLEAN'"
+                   + "            WHEN   52 THEN 'BIGINT'"
+                   + "            WHEN   53 THEN 'BIGSERIAL'"
+                   + "            WHEN 2061 THEN 'IDSSECURITYLABEL'"
+                   + "            WHEN 4118 THEN 'ROW'"
+                   + "        END"
+                   + "           AS data_type"
+                   + "     , " + (forProcedureParameters ? "c.paramattr" : "1") // Dummy value for table/view columns.
+                   + "           AS direction"
+                   + "     , c." + (forProcedureParameters ? "paramid" : "colno")
+                   + "           AS position"
+                   + "  FROM informix." + (forProcedureParameters ? "sysprocedures p" : "systables p")
+                   + "     , informix." + (forProcedureParameters ? "sysproccolumns c" : "syscolumns c")
+                   + " WHERE p." + (forProcedureParameters ? "procid" : "tabid") + " = c." + (forProcedureParameters ? "procid" : "tabid")
+                   + "   AND ";
+
+        if (qualifiers.length == 2) {
+            qry += "LOWER(p.owner) = ? AND ";
+        }
+        qry += "LOWER(p." + (forProcedureParameters ? "procname" : "tabname") + ") = ?";
+
+        return qry;
+    }
+
+    public Map<String, DbParameterAccessor> getAllColumns(String tableOrViewName)
+            throws SQLException {
+        String[] qualifiers = NameNormaliser.normaliseName(tableOrViewName).split("\\.");
+        return readIntoParams(qualifiers, columnsOrParamtersQueryText(false, tableOrViewName));
+    }
+
+    private Map<String, DbParameterAccessor> readIntoParams(String[] queryParameters, String query) throws SQLException {
+        try (PreparedStatement dc = currentConnection.prepareStatement(query)) {
+            for (int i = 0; i < queryParameters.length; i++) {
+                dc.setString(i + 1, NameNormaliser.normaliseName(queryParameters[i]));
+            }
+
+            ResultSet rs = dc.executeQuery();
+            Map<String, DbParameterAccessor> allParams = new HashMap<String, DbParameterAccessor>();
+
+            while (rs.next()) {
+                String paramName = defaultIfNull(rs.getString(1), "");
+                String dataType = rs.getString(2);
+                String direction = rs.getString(3);
+                int position = rs.getInt(4);
+                Direction paramDirection = getParameterDirection(direction);
+                if (paramDirection != RETURN_VALUE && paramName.equals("")) {
+                    throw new SQLException("Missing column or procedure parameter name");
+                }
+                DbParameterAccessor dbp = new DbParameterAccessor(paramName,
+                        paramDirection, getSqlType(dataType),
+                        getJavaClass(dataType),
+                        paramDirection == RETURN_VALUE ? -1
+                                : position);
```

Comment:
Agreed. I'll align indentation for all of the parameter args for `new DbParameterAccessor` too.